tests/test_integrated_channels/test_api/test_degreed2/test_views.py

```diff
@@ -0,0 +1,102 @@
+            context=self.enterprise_customer.uuid,
+        )
+        url = reverse('api:v1:degreed2:configuration-detail', args=[self.degreed2_config.id])
+        response = self.client.get(url)
+        data = json.loads(response.content.decode('utf-8'))
+        self.assertEqual(data.get('degreed_base_url'), self.degreed2_config.degreed_base_url)
+        self.assertEqual(data.get('client_id'), self.degreed2_config.client_id)
+        self.assertEqual(data.get('client_secret'), self.degreed2_config.client_secret)
+        self.assertEqual(data.get('degreed_token_fetch_base_url'), self.degreed2_config.degreed_token_fetch_base_url)
+        self.assertEqual(data.get('enterprise_customer'),
+                         str(self.degreed2_config.enterprise_customer.uuid))
+
+    @mock.patch('enterprise.rules.crum.get_current_request')
+    def test_update(self, mock_current_request):
+        mock_current_request.return_value = self.get_request_with_jwt_cookie(
+            system_wide_role=ENTERPRISE_ADMIN_ROLE,
+            context=self.enterprise_customer.uuid,
+        )
+        url = reverse('api:v1:degreed2:configuration-detail', args=[self.degreed2_config.id])
+        payload = {
+            'degreed_base_url': 'http://testing2',
+            'degreed_token_fetch_base_url': 'foobar',
+            'enterprise_customer': ENTERPRISE_ID,
+            'client_id': 'testing',
+            'client_secret': 'secret',
+        }
+        response = self.client.put(url, payload)
+        self.degreed2_config.refresh_from_db()
+        self.assertEqual(self.degreed2_config.degreed_base_url, 'http://testing2')
+        self.assertEqual(self.degreed2_config.client_id, 'testing')
+        self.assertEqual(self.degreed2_config.client_secret, 'secret')
+        self.assertEqual(self.degreed2_config.degreed_token_fetch_base_url, 'foobar')
+        self.assertEqual(str(self.degreed2_config.enterprise_customer.uuid), ENTERPRISE_ID)
+        self.assertEqual(response.status_code, 200)
+
+    @mock.patch('enterprise.rules.crum.get_current_request')
+    def test_patch(self, mock_current_request):
+        mock_current_request.return_value = self.get_request_with_jwt_cookie(
+            system_wide_role=ENTERPRISE_ADMIN_ROLE,
+            context=self.enterprise_customer.uuid,
+        )
+        url = reverse('api:v1:degreed2:configuration-detail', args=[self.degreed2_config.id])
+        payload = {
+            'degreed_base_url': 'http://testingchange',
+            'enterprise_customer': ENTERPRISE_ID,
+        }
+        response = self.client.patch(url, payload)
+        self.degreed2_config.refresh_from_db()
+        self.assertEqual(self.degreed2_config.degreed_base_url, 'http://testingchange')
+        self.assertEqual(response.status_code, 200)
+
+    @mock.patch('enterprise.rules.crum.get_current_request')
+    def test_delete(self, mock_current_request):
+        mock_current_request.return_value = self.get_request_with_jwt_cookie(
```

Comment:
would be great to add a check for non zero count in this test, before issuing the delete